UPGRADE-6.0.md

```diff
@@ -151,6 +151,11 @@ Monolog
  * The `$actionLevel` constructor argument of `Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy` has been replaced by the `$inner` one which expects an ActivationStrategyInterface to decorate instead. `Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy` is now final.
  * The `$actionLevel` constructor argument of `Symfony\Bridge\Monolog\Handler\FingersCrossed\HttpCodeActivationStrategy` has been replaced by the `$inner` one which expects an ActivationStrategyInterface to decorate instead. `Symfony\Bridge\Monolog\Handler\FingersCrossed\HttpCodeActivationStrategy` is now final.
 
+Notifier
+--------
+
+ * symfony/slack-notifier: Removed `SlackOptions::channel()`, use `SlackOptions::recipient()` instead.
```

Comment:
This message should have been added with the PR that *introduced* the deprecation. 🤔 

Can you please submit this line as a PR to the 5.2 branch?

Reply:
Here you go: #41309